modules/Router.js

```diff
@@ -19,6 +19,7 @@ class Router extends Component {
     history: object,
     children: routes,
     routes, // alias for children
+    RoutingContext: func.isRequired,
```

Comment:
Implicitly I'm sorta re-using the module name here: https://github.com/rackt/react-router/pull/2383/files#diff-1063d4db6b1a5033d4afb74421a91ef4R31

It's not extremely clever, but it saves a tiny amount of typing.